githubscript/docker.d/worker.yml

```diff
@@ -38,7 +38,34 @@ github_projects:
       staging-android-components:
         allowed_actions: ["release"]
         github_token: { "$eval": "GITHUB_TOKEN_WRITE_ACCESS_STAGING" }
-        github_owner: escapewindow
-        github_repo_name: android-components
+        github_owner: mozilla-releng
+        github_repo_name: staging-android-components
+        contact_github: true
+    'COT_PRODUCT == "xpi" && ENV == "prod"':
+      xpi-manifest:
+        allowed_actions: ["release"]
+        github_token: { "$eval": "GITHUB_TOKEN_WRITE_ACCESS_PROD" }
+        github_owner: mozilla-extensions
+        github_repo_name: xpi-manifest
+        contact_github: true
+    'COT_PRODUCT == "xpi" && ENV == "fake-prod"':
+      staging-xpi-manifest:
+        allowed_actions: ["release"]
+        github_token: { "$eval": "GITHUB_TOKEN_WRITE_ACCESS_STAGING" }
+        github_owner: mozilla-releng
+        github_repo_name: staging-xpi-manifest
+        contact_github: true
+    'COT_PRODUCT == "xpi" && ENV == "dev"':
+      staging-xpi-manifest:
+        allowed_actions: ["release"]
+        github_token: { "$eval": "GITHUB_TOKEN_WRITE_ACCESS_STAGING" }
+        github_owner: mozilla-releng
+        github_repo_name: staging-xpi-manifest
         contact_github: true
-taskcluster_scope_prefixes: ["project:mobile:fenix:releng:github:", "project:mobile:android-components:releng:github:"]
+
+scopes:
```

Comment:
I think you're moving `taskcluster_scope_prefixes` from a top level config item to a `scopes.taskcluster_scope_prefixes` item.

I think this should look more like

```
taskcluster_scope_prefixes:
  $switch:
    'COT_PRODUCT == "mobile"':
        [...]
```